variables.yaml.example

```diff
@@ -1,39 +1,41 @@
+# vim: set filetype=yaml :
```

Comment:
🤔

Reply:
This is valid in GitHub and syntax highlights the variables.yaml.example file :wink:

Reply:
👍 I did not know that.